stubs/DynamicFormsResourceController.stub

```diff
@@ -0,0 +1,140 @@
+<?php
+
+
+namespace App\Http\Controllers;
+
+
+use Illuminate\Routing\Controller;
+use Illuminate\Support\Str;
+use Northwestern\SysDev\DynamicForms\ResourceRegistry;
+use Northwestern\SysDev\DynamicForms\Resources\ResourceInterface;
+use Illuminate\Http\Request;
+use Symfony\Component\Finder\Finder;
+
+class ResourceController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    /**
+     * Provides the index of all registered Resources
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(ResourceRegistry $resourceRegistry) //register
+    {
+        $resourceIndexList = array_keys($resourceRegistry->registered());
+
+        $retString = '[';
+        foreach ($resourceIndexList as $name)
+        {
+            $retString .= '{"_id":"'.$name.'","title":"'.$name.'"},';
```

Comment:
This sort of operation is best handled by `json_encode()`. Then there's no need to worry about commas and escaping things when using the built-in encoder.

I would suggest making the resource interface return more arrays instead of JSON strings. Then the resource interface's methods can do arrays and allow the controller to turn them into appropriate JSON.